index.js

```diff
@@ -150,7 +150,7 @@ function parse(input, options) {
 	}
 
 	for (const param of input.split('&')) {
-		let [key, value] = param.replace(/\+/g, ' ').split('=');
+		let [key, value] = param.replace(/\+/g, ' ').replace('=', '<**>').split('<**>');
```

Comment:
Just ran into the same issue. This is the alternative I used for a fix https://github.com/cam-perry/query-string/commit/b65a53b55ae7a8825d62af757a587d365ba7e3b7:
```js
const i = param.indexOf("=");
let key = param.slice(0, i);
let value = param.slice(i+1);
```